types/openui5/sap.ui.core.d.ts

```diff
@@ -87638,6 +87643,8 @@ declare namespace sap {
 
     "sap/ui/core/ComponentContainer": undefined;
 
+    "sap/ui/core/ComponentHooks": undefined;
```

Comment:
We should check whether some filtering is missing for the content of the dependencies interface. sap/ui/core/ComponentHooks only contains restricted APIs and therefore should not occur in this interface.